mpbb-list-subports

```diff
@@ -48,7 +48,7 @@ print-subports() {
         # $thisdir is set in mpbb
         # shellcheck disable=SC2154
         archive_path=$("${tclsh}" "${thisdir}/tools/archive-path.tcl" "${port}")
-        if [[ -f "${archive_path}" ]]; then
+        if [[ -f "${archive_path}" && -n "${archive_site}" ]]; then
```

Comment:
Couldn't we also skip `archive-path.tcl` if `archive_site` is empty?

Reply:
Yes, but I'd like to make a rather small change. Feel free to push a better version instead.

Reply:
I pushed a new commit to the pull request branch. That seemed easier than trying to explain what I meant, which was only to move the conditional.